### DISCUSSION TITLE
Transfer account to phone?

### DISCUSSION THREAD

**Discussion by notourist** [AsamK/signal-cli#1299] [Q&A] [answered]
Hello,

is it possible to transfer the account to a phone with the Signal app after registering it with signal-cli? I have only found the documentation in the wiki regarding linking secondary devices.

**Comment by AsamK** [comment, marked as answer]
No, transferring the account is not possible.

**Reply by ruario** [reply]
Planned for the future?

**Comment by Lastofwithes** [comment]
Nooooo !
It's imposible !
